apps/performance-service/cypress/shared/utils/authorization.ts

```diff
@@ -0,0 +1,24 @@
+export const createAuthorizationHeader = (cypressEnv: string) =>
```

Comment:
I tried putting some of the cypress functions into the shared repo package but struggled to get it to work because using cypress variables like `Cypress`, `cy.`, and `it` don't really work outside of the cypress configured environment. I tried a few different things and then decided I was spending a lot of time on it so I just moved on and copied the code into the performance service.